app/backend/wells/migrations/0083_merge_20190510_2215.py

```diff
@@ -0,0 +1,14 @@
+# Generated by Django 2.2.1 on 2019-05-10 22:15
```

Comment:
I think this file is redundant and will need another merge migration if it's added to release.  Can you try merging in release into this branch?

Reply:
Afraid it's not easy - there's a whole string of dependancies to it now.